templates/person/show.json.twig

```diff
@@ -0,0 +1,51 @@
+{
+    "@context": "https://schema.org",
+    "@graph": [
+        {
+            "@type": "Person",
+            "givenName": "{{ person.firstName}}",
+            "familyName":"{{ person.lastName}}",
+            "sameAs":[
+                "{{ person.wikipediaUrl}}",
+                "{{ person.viafUrl}}"
+            ],
+            "birthDate":"{{person.dob}}",
+            "deathDate":"{{person.dob}}",
+            "birthPlace":"{{person.cityOfBirth}}",
+            "deathPlace":"{{person.cityOfDeath}}",
+            "@id": "{{ person.id }}"
```

Comment:
Should the `@id` also be a full URL (like book, firm)?

Reply:
Yes I changed it I forgot to commit this!